### ISSUE TITLE
Rustix 0.38.22 breaks my build

### ISSUE THREAD

**Issue by holtgrewe** [bytecodealliance/rustix#930]
I'm trying to compile [this app](https://github.com/bihealth/mehari) on branch `main`/`973ab7b`. The program compiled on CI 10h ago but locally I get:

```
error[E0432]: unresolved import `crate::backend::vdso_wrappers`
  --> /home/holtgrem_c/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.22/src/backend/linux_raw/process/syscalls.rs:46:32
   |
46 | pub(crate) use crate::backend::vdso_wrappers::sched_getcpu;
   |                                ^^^^^^^^^^^^^ could not find `vdso_wrappers` in `backend`
```

Removing `target/`, `~/.cargo` and `~/.rustup` and reinstalling did not help.

The following in `Cargo.toml` fixes the issue:

```
rustix = "=0.38.21"  # pin
```

I'm on `Ubuntu 22.04.3 LTS`

```
uname -a
Linux $HOSTNAME 6.2.0-36-generic #37~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Oct  9 15:34:04 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by matthiasbeyer** [comment]
Same here, this just broke a build for me.

**Comment by matthiasbeyer** [comment]
Thanks for the fast fix!
Hope there will be a new patchrelease soon! :tada: :heart:

**Comment by sunfishcode** [comment]
Sorry for the breakage! I've now released rustix 0.38.23 with the fixes, and yanked 0.38.22.
